less/about/preferences.less

```diff
@@ -1561,56 +1557,74 @@ table.sortableTable {
   }
 }
 
-.syncOverlay {
-  input {
-    margin-left: 20px;
-    display: inline;
+.syncContainer {
+  .modal {
+    textarea {
+      width: 80%;
+      height: 100px;
+      font-size: 18px;
+      font-family: monospace;
+    }
+    li {
+      margin: 1em;
+    }
+    .settingsListContainer {
+      margin-bottom: 1.5em;
+    }
+    .dialog-body, .dialog-footer {
+      padding-left: 50px;
+      padding-right: 50px;
+    }
+    .dialog-footer {
+      margin-top: 1.5em;
+      padding-bottom: 2em;
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: flex-start;
+    }
   }
-  textarea {
-    width: 80%;
-    height: 100px;
-    font-size: 18px;
-    font-family: monospace;
+  .browserButton + .browserButton {
+    margin-left: 0.75em;
```

Comment:
I think this should be covered with https://github.com/brave/browser-laptop/blob/2aa39bb1073fe8bb40d9fc32d18fdd2ea9cee1d5/less/button.less#L52